pkg/signatures/cosign_sig_fetcher.go

```diff
@@ -111,13 +112,27 @@ func (c *cosignSignatureFetcher) FetchSignatures(ctx context.Context, image *sto
 				fullImageName, err)
 			continue
 		}
+
+		certPEM, err := certificateFromSignedPayload(signedPayload)
+		if err != nil {
+			log.Errorf("Error during unmarshalling certificate to PEM for image %q: %v", fullImageName, err)
+		}
+
+		chainPEM, err := certificateChainFromSignedPayload(signedPayload)
+		if err != nil {
+			log.Errorf("Error during unmarshalling certificate chain to PEM for image %q: %v",
+				fullImageName, err)
+		}
+
 		// Since we are only focusing on public keys, we are ignoring the certificate / rekor bundles associated with
```

Comment:
nit:  do we need any adjustments to this comment now that we are including some certificates?